src/dlgPackageExporter.cpp

```diff
@@ -381,6 +382,8 @@ bool dlgPackageExporter::eventFilter(QObject* obj, QEvent* evt)
             for (int i = mDescriptionImages.size() - 1; i >= 0; i--) {
                 QString fname = mDescriptionImages.at(i);
                 QFileInfo info(fname);
+                fname = QUrl::toPercentEncoding(fname).constData();
```

Comment:
This works well - results in `fname is: "%2FUsers%2Fadministrator%2FDownloads%2Fpicture%20with%20spaces%20in%20name.jpg"
` @Edru2